src/v/kafka/server/group_tx_tracker_stm.h

```diff
@@ -133,12 +134,14 @@ class group_tx_tracker_stm final
     }
 
 private:
-    static constexpr int8_t supported_local_snapshot_version = 1;
+    static constexpr int8_t supported_local_snapshot_version = 2;
     struct snapshot
-      : serde::envelope<snapshot, serde::version<0>, serde::compat_version<0>> {
+      : serde::envelope<snapshot, serde::version<1>, serde::compat_version<0>> {
         all_txs_t transactions;
+        chunked_vector<kafka::group_id> blocked_groups;
```

Comment:
oh I see this was added here, perhaps squash this commit 79085150a730057104e2e67518878e0ec64c1a68

Reply:
do you mean combine f8f1341844b3312c03be0ff5e691dccf96229a31 and 79085150a730057104e2e67518878e0ec64c1a68 into a single commit?